index.html

```diff
@@ -4,10 +4,312 @@
+                </a>
+              </div>
+            </div>
+          </div>
+          <div class="row align-items-center">
+            <div class="col-8 offset-2">
+              <div class="steps__line">
+                <svg
+                  class="line"
+                  viewbox="0 0 874 92"
+                  fill="none"
+                  xmlns="http://www.w3.org/2000/svg"
+                >
+                  <path
+                    opacity="0.502"
+                    d="M1 91C1 91 217.014 -110.278 442.118 91C442.118 91 639.939 -111.5 873 91"
+                    stroke="white"
+                    stroke-linecap="round"
+                    stroke-dasharray="2 5"
+                  />
+                </svg>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="travel-ad-up travel" id="what_we_do">
+        <div class="container">
+          <div class="row">
+            <div class="col-12 col-lg-7 col-xl-6">
+              <div class="travel-ad-up__card card">
+                <div class="card__title--small">
+                  <p class="title--small">spain</p>
+                </div>
+                <div class="card__title">
+                  <h5 class="title-5">Grand Spain Madrid</h5>
+                </div>
+                <div class="card__text text">
+                  But I must explain to you how all this mistaken idea of
+                  account to denouncing pleasure and praising pain was account
+                  born and I account will give you a complete account of the
+                  system
+                </div>
+                <div class="card__button button">book now</div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="travel-ad-down travel" id="travel">
+        <div class="container">
+          <div class="row">
+            <div class="col-12 col-lg-7 offset-lg-5 col-xl-6 offset-xl-6">
+              <div class="travel-ad-down__card card">
+                <div class="card__title--small">
+                  <p class="title--small">spain</p>
+                </div>
+                <div class="card__title">
+                  <h5 class="title-5">Grand Spain Madrid</h5>
+                </div>
+                <div class="card__text text">
+                  But I must explain to you how all this mistaken idea of
+                  account to denouncing pleasure and praising pain was account
+                  born and I account will give you a complete account of the
+                  system
+                </div>
+                <div class="card__button button">book now</div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="what">
+        <div class="what__title">
+          <h2 class="title-2">What Our Tourists Say</h2>
+        </div>
```

Comment:
Не помилка, але я думаю, що чим менше коду, тим краще=)
```suggestion
        <h2 class="title-2 what__title">What Our Tourists Say</h2>
```